x-pack/plugins/file_upload/public/util/http_service.js

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+
+
+// service for interacting with the server
```

Comment:
this is a copy-paste of the ML-code https://github.com/thomasneirynck/kibana/blob/277634507fa16d5b77b39bf652d2162df5652702/x-pack/plugins/ml/public/services/http_service.js#L15

remove this from this from this PR and add to it to https://github.com/elastic/kibana/pull/36404

Reply:
We discussed offline, but for documentation: I'm keeping this a part of this PR since it's a UI concern. The PR you're referencing is strictly server-side and could all be used by ML. This service would be unlikely to be leveraged by ML unless we started moving their UI logic into this plugin in the future.